docs/modules/ROOT/pages/index.adoc

```diff
@@ -392,11 +403,11 @@ sequenceDiagram
 
 IMPORTANT: Validation of the _<<C2PA Manifest>>_ MUST be completed with a finding that the manifest is at least _well-formed_ as per link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_well_formed_manifest++[Section 14.3.2, Well-formed manifest,”] of the C2PA technical specification before a validator attempts to report on the validity of an *<<_identity_assertion,identity assertion>>.*
 
-An *<<_identity_assertion,identity assertion>>* MUST contain valid CBOR that satisfies the `identity` rule in xref:_schema_and_example[xrefstyle=full]. The `cawg.identity.cbor.invalid` error code SHALL be used to report assertions that do not follow this rule.
+An *<<_identity_assertion,identity assertion>>* MUST contain valid CBOR that satisfies the `identity` rule in xref:_cbor_schema[xrefstyle=full]. The `cawg.identity.cbor.invalid` error code SHALL be used to report assertions that do not follow this rule.
```

Comment:
C2PA very specifically does *NOT* require schema validation as part of the Manifest validation process.  It is unclear if you are suggesting here that identity assertions would require it - and if so, please don't.

Reply:
Change this to say "structurally valid" CBOR (see "canonical CBOR" language in CBOR spec 3.9 of RFC 7049) and contains all required fields as described in `identity` rule. (Verifiers should ignore extra fields not specified.)

Reply:
Since this isn't specific to the `tbs` data structure, I moved this out to its own PR #81.